geopandas/tests/test_clip.py

```diff
@@ -291,17 +291,18 @@ def test_clip_with_polygon(single_rectangle_gdf):
     assert hasattr(clip, "geometry") and clip.geom_type[0] == "Polygon"
 
 
-def test_clip_with_line_sliver(single_rectangle_gdf, sliver_line):
-    """Test clipping a line so that there is a sliver drops the sliver."""
-    clip = gpd.clip(sliver_line, single_rectangle_gdf, drop_slivers=True)
+def test_clip_with_line_extra_geom(single_rectangle_gdf, sliver_line):
+    """Test clipping a line so that there is an extra geometry type generated."""
```

Comment:
```suggestion
    """When the output of a clipped line returns a geom collection, and geek_geom_type is True, no geometry collections should be returned"""
```
@nkorinek this line length will be too long.